### PR TITLE
PCSUP-5834

### PR DESCRIPTION
**Pull request by Pubs-MV**
updates for TLS interception with LB ( and proxy)

### CODE REVIEW

**Comment by sgordon46** [comment]
This looks good to me.

File: compute/admin_guide/configure/proxy.adoc [hunk 1]
```diff
@@ -28,7 +28,9 @@ Defender has a number of connections that might traverse a proxy.
 image::configure_proxy_defender.png[width=500]
 
 * Connecting to Console.
-If you deploy Defenders in a remote region, they might need to connect to Console through a proxy.
++
+If you have a proxy or a load balancer between Defender and Console, make sure that TLS interception is not enabled. The certificate and keys used for the Console to Defender mutual TLS v1.2 web socket session are generated when the Console is initiated. This ensures that the Console is only communicating with the Defenders it has deployed and the Defenders only communicate with the Console that manages it. While you can use CA certificates for Console and Defender communications to external systems, the Defender to Console communication cannot be TLS intercepted.
```
Comment (rcaballeromx) [inline compute/admin_guide/configure/proxy.adoc]:
```suggestion
If you have a proxy or a load balancer between Defender and Console, make sure that TLS interception is not enabled. The certificate and keys used for the Console to Defender mutual TLS v1.2 web socket session are generated when the Console is initiated. This ensures that the Console is only communicating with the Defenders it has deployed and the Defenders only communicate with the Console that manages it. While you can use CA certificates for the communications from Console and Defender to external systems, the Defender to Console communication cannot be TLS intercepted.
```

**Review by rcaballeromx — COMMENTED** [review]
Only one suggestion based on our conversation today.

**Comment by sgordon46** [comment]
@rcaballeromx @Pubs-MV good to merge.

**Comment by pfox1969** [comment]
looks good.

**Review by pfox1969 — APPROVED** [review]

**Pull request merged**